### PR TITLE
Fixes in testGCIntegrationWithRedisFork, testDelReplicate

### PR DESCRIPTION
**Pull request by rafie**

### CODE REVIEW

**Comment by MeirShpilraien** [comment]
@rafie as we talked on slack, this is not the correct fix..

File: src/pytest/test_replicate.py [hunk 1]
```diff
@@ -41,9 +42,9 @@ def testDelReplicate():
   env.skipOnCluster()
 
   master = env.getConnection()
-  slave = env.getSlaveConnection() 
-  env.assertContains("PONG", master.execute_command("ping"))
-  env.assertContains("PONG", slave.execute_command("ping"))  
+  slave = env.getSlaveConnection()
+  env.assertTrue(master.execute_command("ping"))
```
Comment (MeirShpilraien) [inline src/pytest/test_replicate.py]: @rafie how did it pass without those changes?

Comment (rafie) [reply src/pytest/test_replicate.py]: I think there is some output processing on the part of redis-py in those cases, so it's probably related to the 3.0 <-> 3.4 version differences.

Comment (MeirShpilraien) [reply src/pytest/test_replicate.py]: Is this PR changes the version we are installing?

Comment (rafie) [reply src/pytest/test_replicate.py]: Yes, we end up with a combination of the latest version of redis-py (3.4.1) with a version of redis-py-cluster (2.1.0) that can handle it (the latter coming from github, not from pypi).
This is currently the state I'm trying to reach in all modules.
The other possibility is redis-py-cluster 2.0 with redis-py 3.0/3.1, which I consider more problematic as it bounds the redis-py version.

Comment (MeirShpilraien) [reply src/pytest/test_replicate.py]: :+1:

**Pull request merged**